main/routines.c

```diff
@@ -149,7 +147,7 @@
 /*
  *  Miscellaneous macros
  */
-#define selected(var,feature)	(((int)(var) & (int)(feature)) == (int)feature)
+
```

Comment:
How do I say... My recognition is that the region from where I removed the macro is a section in which macros are defined.  Though I don't write about the style rule in this project, using two lines for representing the existence of a section is acceptable.

(Sometimes I also made comments about whitespaces. It is basically because git diff reported them with printing them in red.)